build-config/make/mtdutils.make

```diff
@@ -24,7 +24,7 @@ MTDUTILS_STAMP		= $(MTDUTILS_SOURCE_STAMP) \
 			  $(MTDUTILS_BUILD_STAMP) \
 			  $(MTDUTILS_INSTALL_STAMP)
 
-MTDBINS = mkfs.jffs2 mkfs.ubifs ubinize ubiformat ubinfo mtdinfo
+MTDBINS = mkfs.jffs2 mkfs.ubifs ubinize ubiformat ubinfo mtdinfo ubiattach ubimkvol ubidetach ubirmvol
```

Comment:
The busybox install comes after so what I found was if I have UBI enabled in the busybox config then the busybox utilities will still "rule" and be installed in the sysroot by preference (it would overwrite the mtdutils version).
I removed the UBI config from my boards busybox config to allow the mtdutils version to be the only ones. 
I can remove UBI config from the default config if you want - I was assuming that would potentially impact more boards so wouldn't be a good plan :)